projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,582 @@
+	"advancements.projectexpansion.cyan_fuel": "青燃料",
+	"advancements.projectexpansion.cyan_fuel.description": "获得青燃料。",
+	"advancements.projectexpansion.cyan_fuel_block": "青燃料块",
+	"advancements.projectexpansion.cyan_fuel_block.description": "获得青燃料块。",
+	"advancements.projectexpansion.green_fuel": "绿燃料",
+	"advancements.projectexpansion.green_fuel.description": "获得绿燃料。",
+	"advancements.projectexpansion.green_fuel_block": "绿燃料块",
+	"advancements.projectexpansion.green_fuel_block.description": "获得绿燃料块。",
+	"advancements.projectexpansion.lime_fuel": "黄绿燃料",
+	"advancements.projectexpansion.lime_fuel.description": "获得黄绿燃料。",
+	"advancements.projectexpansion.lime_fuel_block": "黄绿燃料块",
+	"advancements.projectexpansion.lime_fuel_block.description": "获得黄绿燃料块。",
+	"advancements.projectexpansion.yellow_fuel": "黄燃料",
+	"advancements.projectexpansion.yellow_fuel.description": "获得黄燃料。",
+	"advancements.projectexpansion.yellow_fuel_block": "黄燃料块",
+	"advancements.projectexpansion.yellow_fuel_block.description": "获得黄燃料块。",
+	"advancements.projectexpansion.orange_fuel": "橙燃料",
+	"advancements.projectexpansion.orange_fuel.description": "获得橙燃料。",
+	"advancements.projectexpansion.orange_fuel_block": "橙燃料块",
+	"advancements.projectexpansion.orange_fuel_block.description": "获得橙燃料块。",
+	"advancements.projectexpansion.white_fuel": "白燃料",
+	"advancements.projectexpansion.white_fuel.description": "获得白燃料。",
+	"advancements.projectexpansion.white_fuel_block": "白燃料块",
+	"advancements.projectexpansion.white_fuel_block.description": "获得白燃料块。",
+	"advancements.projectexpansion.fading_fuel": "渐消燃料",
+	"advancements.projectexpansion.fading_fuel.description": "获得渐消燃料。",
+	"advancements.projectexpansion.fading_fuel_block": "渐消燃料块",
+	"advancements.projectexpansion.fading_fuel_block.description": "获得渐消燃料块。",
+	"advancements.projectexpansion.magnum_star_omega": "更大的EMC电池？？",
+	"advancements.projectexpansion.magnum_star_omega.description": "应该没有人拥有这么多的能量。",
+	"advancements.projectexpansion.colossal_star_omega": "为什么你需要这么多EMC？",
+	"advancements.projectexpansion.colossal_star_omega.description": "这越来越荒谬了。",
+	"advancements.projectexpansion.final_star_shard": "你玩这个游戏太久了",
+	"advancements.projectexpansion.final_star_shard.description": "说真的，出去走走吧。",
+	"advancements.projectexpansion.transmutation_interface": "电脑里的转化？？",
+	"advancements.projectexpansion.transmutation_interface.description": "它们变得更加出色了。",
+	"advancements.projectexpansion.infinite_steak": "无限牛排！！！",
+	"advancements.projectexpansion.infinite_steak.description": "面包？那是为农民准备的。",
+	"advancements.projectexpansion.infinite_fuel": "是恒星吗？",
+	"advancements.projectexpansion.infinite_fuel.description": "那是一个十字架吗？管它呢，扔进熔炉里看它燃烧。",
+	"advancements.projectexpansion.collector": "更强的能量收集器",
+	"advancements.projectexpansion.collector.description": "现在荒谬的事情开始了。",
+	"advancements.projectexpansion.compressed_collector": "嘿，看我把收集器压缩成了什么！",
+	"advancements.projectexpansion.compressed_collector.description": "现在它们暂时没用了，除非它们被制作成力量花盆栽罐。",
```

```suggestion
	"advancements.projectexpansion.compressed_collector.description": "现在这玩意儿暂时没用了，除非被制作成别的东西。",
```
个人观点，可能有点口语化
应当以原文为准，原文没说是做盆栽，文本与游戏内容一致的情况下，就不要加戏